tests/repl.sh

```diff
@@ -71,3 +73,48 @@ testReplResponse '
 :a { a = "2"; }
 "result: ${a}"
 ' "result: 2"
+
+testReplResponse '
+drvPath
+' '".*-simple.drv"' \
+$testDir/simple.nix --experimental-features ''
```

Comment:
```suggestion
$testDir/simple.nix
```

(Otherwise the CA tests will fail because they require the `ca-derivations` xp feature)

And likewise below